packages/xo-web/src/common/intl/messages.js

```diff
@@ -1230,8 +1230,8 @@ const messages = {
   srFormat: 'Type',
   srSize: 'Size',
   srUsage: 'Usage',
-  srUsed: 'used',
-  srFree: 'free',
+  srUsed: 'Used',
+  srFree: 'Free',
```

Comment:
Does it make sense to be capitalized when it's used after a size, like `5 GiB Free`?

Reply:
I'm not sure exactly, I guess it does depend on context. My understanding was that we wanted to standardize on capitalization method across all messages

Reply:
If we don't know about the "usage", we have to check which choice has the current majority in XO (lower case of upper case after an unit), then use the same for the whole software.
In terms of "feeling" (and not in XO only), I think I've seen most lower case for usage/free, but I'm not sure.

Reply:
After some thinking, it would be probably better to use lowercase thing because it's more logical when it's not on the start of a sentence. As soon this is changed, we could merge this :)

Reply:
The issue is that we should not have translated like this, we should have used a variable: `{bytes} free`.

To be properly translatable, it should always be a complete string, not a piece of a sentence.

/cc @pdonias

Reply:
in the meantime I've changed them back to lowercase